frame/oracle/src/tests.rs

```diff
@@ -1298,14 +1304,13 @@ fn historic_pricing() {
 #[test]
 fn price_of_amount() {
 	new_test_ext().execute_with(|| {
-		let value = 100500;
-		let id = 42;
-		let amount = 10000;
+		// Yes I know this is broken currently
```

Comment:
Would prefer a comment describing why it is broken :)

Reply:
Part of the issue is I'm not sure why it's broken - I'm going to take another crack at it later and see if I can figure it out

Reply:
See https://github.com/ComposableFi/composable/pull/1207/commits/61b671de32eeaef39511196eb244971313ac73a2, still not sure what the issue is

Reply:
Added a proptest (https://github.com/ComposableFi/composable/pull/1207/commits/231b0d9e4570c102f34e4ac94eae2d17096633ac), every input fails. I think the issue is https://github.com/ComposableFi/composable/blob/main/frame/oracle/src/lib.rs#L1010-L1018 ? I'm not sure what this function is for/ what it's expected inputs & outputs are; it's only used [here](https://github.com/ComposableFi/composable/blob/982c67b19051461d8583d46088c7d43679f22082/frame/oracle/src/lib.rs#L419) and [here](https://github.com/ComposableFi/composable/blob/982c67b19051461d8583d46088c7d43679f22082/frame/oracle/src/lib.rs)